src/sentry/integrations/discord/message_builder/issues.py

```diff
@@ -0,0 +1,149 @@
+from __future__ import annotations
+
+from sentry import tagstore
+from sentry.eventstore.models import GroupEvent
+from sentry.integrations.discord.message_builder import LEVEL_TO_COLOR
+from sentry.integrations.discord.message_builder.base.base import DiscordMessageBuilder
+from sentry.integrations.discord.message_builder.base.component.action_row import DiscordActionRow
+from sentry.integrations.discord.message_builder.base.component.base import DiscordMessageComponent
+from sentry.integrations.discord.message_builder.base.component.button import DiscordButton
+from sentry.integrations.discord.message_builder.base.embed.base import DiscordMessageEmbed
+from sentry.integrations.discord.message_builder.base.embed.field import DiscordMessageEmbedField
+from sentry.integrations.discord.message_builder.base.embed.footer import DiscordMessageEmbedFooter
+from sentry.integrations.message_builder import (
+    build_attachment_text,
+    build_attachment_title,
+    build_footer,
+    get_title_link,
+    has_releases,
+)
+from sentry.models.group import Group, GroupStatus
+from sentry.models.project import Project
+from sentry.models.rule import Rule
+from sentry.notifications.notifications.base import ProjectNotification
+from sentry.types.integrations import ExternalProviders
+
+from ..message_builder.base.component import DiscordComponentCustomIds as CustomIds
+
+
```

Comment:
This is a wrapper around DiscordMessageBuilder that handles turning the issue alert into a pretty Discord message.